data_steward/cleaners/clean_rule_4.py

```diff
@@ -0,0 +1,52 @@
+"""
+Rule: 4
+ID columns in each domain should be unique
+"""
+
+import bq_utils
+import cdm
+import clean_cdr_engine
+import resources
+
+rule_4_query = """
```

Comment:
Make this a descriptive name.  For example, `ensure_unique_ids`.   I suggest updating the name of the module too.  When looking at files in a directory, no one is going to know what `clean_rule_4.py` is supposed/actually doing.